api/python/quilt3/data_transfer.py

```diff
@@ -26,6 +27,7 @@
 
 
 MAX_COPY_FILE_LIST_RETRIES = 3
+use_tqdm = os.getenv('QUILT_USE_TQDM').lower() == 'true'
```

Comment:
1. Use all caps for constants
2. `.lower()` will fail if `None` is returned :(
3. `== 'true'` is super fragile; just use the truthiness of the string
```suggestion
USE_TQDM = os.getenv('QUILT_USE_TQDM', '').lower()
```

Reply:
> 3. `== 'true'` is super fragile; just use the truthiness of the string

In this case, the truthiness of the string might not work since the string is passed as an env variable and can be any value. I had to explicitly compare to "true" to mitigate that.

<img width="549" alt="Screenshot 2020-03-20 at 8 03 22 AM" src="https://user-images.githubusercontent.com/30266850/77143127-8c475800-6a82-11ea-9153-28f93d954d8d.png">

Reply:
[Here's](https://github.com/quiltdata/quilt/blob/22c0c18fc86ce1f7b002c63fdf373db76fa0a4ce/api/python/quilt3/telemetry.py#L66) a similar usage for `QUILT_DISABLE_USAGE_METRICS`

Reply:
You can still check `USE_TQDM == "whatever"` but you have to provide a default value or else that Code will crash for undefined varaibles.

Reply:
So for instance your code `os.getenv('QUILT_USE_TQDM').lower()` will crash if the environment variable is undefined.

Reply:
Yes, sure. I will update that change request.